tslint.d.ts doesn't provide an external module

I'm writing a custom rule in TypeScript, so I need to provide tsc with the typings for superclasses such as `Lint.Rules.AbstractRule`.
The distributed .d.ts file has internal modules, eg.
```ts
declare module Lint.Rules {
}
```
So when I try the obvious thing,
```ts
/// <reference path="../../node_modules/tslint/lib/tslint.d.ts" />
export class Rule extends Lint.Rules.AbstractRule {
}
```
I get error
`tslint/functionHasReturnTypeRule.ts(7,27): error TS2503: Cannot find namespace 'Lint'.`

IIUC the .d.ts also needs to re-export as an external module, like 
```ts
declare module "tslint" {
  export = Lint;
}
```

--- a/lib/tslint.d.ts
+++ b/lib/tslint.d.ts
@@ -258,3 +258,6 @@ declare module Lint {
         private containsRule(rules, rule);
     }
 }
+declare module "tslint" {
+    export = Lint;
+}

--- a/src/tslint.ts
+++ b/src/tslint.ts
@@ -110,3 +110,7 @@ global.ts = ts;
 module.exports = Lint.Linter;
 // also export Lint.Configuration.findConfiguration so implementers can consume
 module.exports.findConfiguration = Lint.Configuration.findConfiguration;
+
+declare module "tslint" {
+    export = Lint;
+}

